README.md

```diff
@@ -201,6 +206,9 @@ You can disable `LOG_XXXX` and `ASSERT` macro completely by defining following m
 // You can also set default log level by defining macro (default: INFO)
 // #define DEBUGLOG_DEFAULT_LOG_LEVEL_TRACE
 
+// You can call other macros, functions, or define strings that will be included in all log statements
+#define LOG_PREAMBLE __FILE__ " : "
+
```

Comment:
[nit] I prefer to write examples of PREAMBLE configurations with various patterns :) (please note that I may modify this myself later!)

Reply:
How about I provide examples of combinations of:

- [ ] Strings
- [ ] Macros
- [ ] Function Calls

Would these three examples be enough?